node_modules/col-categories/tests/util.js

```diff
@@ -19,12 +19,15 @@ var assert = require('assert');
 /**
  * Assert that a category has all expected properties
  *
- * @param  {Category}           category                The category to assert the properties for
- * @param  {Category}           [expectedCategory]      The category to which the provided category should be compared
- * @param  {Boolean}            [expectAssetCount]      Whether the number of assets associated to the category are expected to be included
- * @throws {AssertionError}                             Error thrown when an assertion failed
+ * @param  {Category}           category                      The category to assert the properties for
+ * @param  {Object}             [opts]                        Optional parameters to verify the category with
+ * @param  {Category}           [opts.expectedCategory]       The category to which the provided category should be compared
+ * @param  {Boolean}            [opts.expectAssetCount]       Whether the number of assets associated to the category are expected to be included
+ * @throws {AssertionError}                                   Error thrown when an assertion failed
  */
```

Comment:
@nicolaasmatthijs I assume this change is so you can pass in multiple expectations down the road?